### DISCUSSION TITLE
Help adding unsupported NEO device

### DISCUSSION THREAD

**Discussion by kasi0815** [Koenkk/zigbee2mqtt#7268] [Devices]
Hi, it is the first time that I ask for help for this topic. Please bear with me if I do not yet adhere to a certain form. I bought a temperature sensor model NAS-TH02B manufactured by Neo. According the short manual it should connect to the Tuya app "Smart Life". Because this device is not supported by zigbee2mqtt yet, I added the following part to the devices.js:

`    {
        fingerprint: [
            {type: 'EndDevice', manufacturerID: 4098, endpoints: [{ID: 1, inputClusters: [], outputClusters: []}]},
            {manufacturerName: '_TZ3000_qaaysllp'},
        ],
        zigbeeModel: ['TS0201'],
        model: 'TS0201',
        vendor: 'NEO',
        description: 'Temperature, humidity & illuminance sensor with display',
        fromZigbee: [fz.battery, fz.temperature, fz.humidity, fz.illuminance],
        toZigbee: [],
        exposes: [e.battery(), e.temperature(), e.humidity(), e.illuminance()],
    },
`

If I pair it now with my CC26x2R LaunchPad than it communicates the battery status and the illuminance like this:

`{"battery":100,"device":{"applicationVersion":67,"dateCode":"","friendlyName":"KlimaTuya01","hardwareVersion":1,"ieeeAddr":"0x842e14fffee547f0","manufacturerID":4098,"manufacturerName":"_TZ3000_qaaysllp","model":"TS0201","networkAddress":33629,"powerSource":"Battery","stackVersion":0,"type":"EndDevice","zclVersion":3},"illuminance":11762,"illuminance_lux":15,"linkquality":12}`

The values are coorect. However there is also temperature and humidity measured but not shown on mqtt. I went through the "How to support new devices" procedure including for Tuya devices but I stuck. 

In the following debug log I saw that there is maybe more data:
```
Client mosqsub|22730-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/logging', ... (202 bytes))
{"level":"info","message":"MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{\"data\":{\"friendly_name\":\"KlimaTuya01\",\"ieee_address\":\"0x842e14fffee547f0\"},\"type\":\"device_announce\"}'"}
Client mosqsub|22730-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/event', ... (101 bytes))
{"data":{"friendly_name":"KlimaTuya01","ieee_address":"0x842e14fffee547f0"},"type":"device_announce"}
Client mosqsub|22730-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/logging', ... (186 bytes))
{"level":"info","message":"MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{\"message\":\"announce\",\"meta\":{\"friendly_name\":\"KlimaTuya01\"},\"type\":\"device_announced\"}'"}
Client mosqsub|22730-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/log', ... (87 bytes))
{"message":"announce","meta":{"friendly_name":"KlimaTuya01"},"type":"device_announced"}
Client mosqsub|22730-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/logging', ... (186 bytes))
{"level":"debug","message":"Received Zigbee message from 'KlimaTuya01', type 'attributeReport', cluster 'msIlluminanceMeasurement', data '{\"61441\":50}' from endpoint 1 with groupID 0"}
Client mosqsub|22730-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/logging', ... (158 bytes))
{"level":"debug","message":"No converter available for 'TS0201' with cluster 'msIlluminanceMeasurement' and type 'attributeReport' and data '{\"61441\":50}'"}
Client mosqsub|22730-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/logging', ... (186 bytes))
{"level":"debug","message":"Received Zigbee message from 'KlimaTuya01', type 'attributeReport', cluster 'msIlluminanceMeasurement', data '{\"61441\":50}' from endpoint 1 with groupID 0"}
Client mosqsub|22730-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/logging', ... (158 bytes))
{"level":"debug","message":"No converter available for 'TS0201' with cluster 'msIlluminanceMeasurement' and type 'attributeReport' and data '{\"61441\":50}'"}
Client mosqsub|22730-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/logging', ... (196 bytes))
{"level":"debug","message":"Received Zigbee message from 'KlimaTuya01', type 'raw', cluster '57346', data '{\"data\":[8,81,10,10,208,41,40,0],\"type\":\"Buffer\"}' from endpoint 1 with groupID 0"}
Client mosqsub|22730-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/logging', ... (168 bytes))
{"level":"debug","message":"No converter available for 'TS0201' with cluster '57346' and type 'raw' and data '{\"data\":[8,81,10,10,208,41,40,0],\"type\":\"Buffer\"}'"}
Client mosqsub|22730-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/logging', ... (196 bytes))
{"level":"debug","message":"Received Zigbee message from 'KlimaTuya01', type 'raw', cluster '57346', data '{\"data\":[8,82,10,11,208,41,10,0],\"type\":\"Buffer\"}' from endpoint 1 with groupID 0"}
Client mosqsub|22730-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/logging', ... (168 bytes))
{"level":"debug","message":"No converter available for 'TS0201' with cluster '57346' and type 'raw' and data '{\"data\":[8,82,10,11,208,41,10,0],\"type\":\"Buffer\"}'"}
Client mosqsub|22730-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/logging', ... (83 bytes))
{"level":"debug","message":"Saving state to file /opt/zigbee2mqtt/data/state.json"}
Client mosqsub|22730-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/logging', ... (196 bytes))
{"level":"debug","message":"Received Zigbee message from 'KlimaTuya01', type 'raw', cluster '57346', data '{\"data\":[8,83,10,13,208,41,90,0],\"type\":\"Buffer\"}' from endpoint 1 with groupID 0"}
Client mosqsub|22730-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/logging', ... (168 bytes))
{"level":"debug","message":"No converter available for 'TS0201' with cluster '57346' and type 'raw' and data '{\"data\":[8,83,10,13,208,41,90,0],\"type\":\"Buffer\"}'"}
Client mosqsub|22730-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/logging', ... (196 bytes))
{"level":"debug","message":"Received Zigbee message from 'KlimaTuya01', type 'raw', cluster '57346', data '{\"data\":[8,84,10,14,208,41,20,0],\"type\":\"Buffer\"}' from endpoint 1 with groupID 0"}
Client mosqsub|22730-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/logging', ... (168 bytes))
{"level":"debug","message":"No converter available for 'TS0201' with cluster '57346' and type 'raw' and data '{\"data\":[8,84,10,14,208,41,20,0],\"type\":\"Buffer\"}'"}
Client mosqsub|22730-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/logging', ... (192 bytes))
{"level":"debug","message":"Received Zigbee message from 'KlimaTuya01', type 'raw', cluster '57346', data '{\"data\":[8,85,10,6,208,48,2],\"type\":\"Buffer\"}' from endpoint 1 with groupID 0"}
Client mosqsub|22730-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/logging', ... (164 bytes))
{"level":"debug","message":"No converter available for 'TS0201' with cluster '57346' and type 'raw' and data '{\"data\":[8,85,10,6,208,48,2],\"type\":\"Buffer\"}'"}
Client mosqsub|22730-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/logging', ... (193 bytes))
{"level":"debug","message":"Received Zigbee message from 'KlimaTuya01', type 'raw', cluster '57346', data '{\"data\":[8,86,10,15,208,48,2],\"type\":\"Buffer\"}' from endpoint 1 with groupID 0"}
Client mosqsub|22730-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/logging', ... (165 bytes))
{"level":"debug","message":"No converter available for 'TS0201' with cluster '57346' and type 'raw' and data '{\"data\":[8,86,10,15,208,48,2],\"type\":\"Buffer\"}'"}
Client mosqsub|22730-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/logging', ... (193 bytes))
{"level":"debug","message":"Received Zigbee message from 'KlimaTuya01', type 'raw', cluster '57346', data '{\"data\":[8,87,10,16,208,32,0],\"type\":\"Buffer\"}' from endpoint 1 with groupID 0"}
Client mosqsub|22730-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/logging', ... (165 bytes))
{"level":"debug","message":"No converter available for 'TS0201' with cluster '57346' and type 'raw' and data '{\"data\":[8,87,10,16,208,32,0],\"type\":\"Buffer\"}'"}
```
The entry in database.db for this device is:
`{"id":31,"type":"EndDevice","ieeeAddr":"0x842e14fffee547f0","nwkAddr":33629,"manufId":4098,"manufName":"_TZ3000_qaaysllp","powerSource":"Battery","modelId":"TS0201","epList":[1],"endpoints":{"1":{"profId":260,"epId":1,"devId":262,"inClusterList":[0,1,1024,57346],"outClusterList":[25,10],"clusters":{"msIlluminanceMeasurement":{"attributes":{"61441":50,"measuredValue":0}},"genBasic":{"attributes":{"modelId":"TS0201","manufacturerName":"_TZ3000_qaaysllp","powerSource":3,"zclVersion":3,"appVersion":67,"stackVersion":0,"hwVersion":1,"dateCode":""}},"genPowerCfg":{"attributes":{"batteryPercentageRemaining":200}}},"binds":[],"configuredReportings":[],"meta":{}}},"appVersion":67,"stackVersion":0,"hwVersion":1,"dateCode":"","zclVersion":3,"interviewCompleted":true,"meta":{},"lastSeen":1619814961378}`

But I do not know how to deal with this raw data type buffer. Is there a documentation which could guide me into the right direction or does somebody can help me?

Many thanks in advance.

**Comment by kasi0815** [comment]
Here is another log part from today after updating z2m to version 1.8.3. The values "illuminance", "illuminance_lux" and "battery" are same as on the device display. By that way, I call the device "KlimaTuya01". I still stuck to retrieve the additional parameters like temperature and humidity. Does somebody has an idea whether temperature and humidity are part of the message or can guide me into the right direction to translate those "raw" data messages?

`{"level":"info","message":"MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{\"data\":{\"friendly_name\":\"KlimaTuya01\",\"ieee_address\":\"0x842e14fffee547f2\"},\"type\":\"device_announce\"}'"}
Client mosqsub|846-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/event', ... (101 bytes))
{"data":{"friendly_name":"KlimaTuya01","ieee_address":"0x842e14fffee547f2"},"type":"device_announce"}
Client mosqsub|846-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/logging', ... (186 bytes))
{"level":"info","message":"MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{\"message\":\"announce\",\"meta\":{\"friendly_name\":\"KlimaTuya01\"},\"type\":\"device_announced\"}'"}
Client mosqsub|846-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/log', ... (87 bytes))
{"message":"announce","meta":{"friendly_name":"KlimaTuya01"},"type":"device_announced"}
Client mosqsub|846-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/logging', ... (192 bytes))
{"level":"debug","message":"No converter available for 'TS0601_temperature_humidity_sensor' with cluster 'manuSpecificTuya' and type 'commandSetTimeRequest' and data '{\"payloadSize\":2816}'"}
Client mosqsub|846-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/logging', ... (186 bytes))
{"level":"debug","message":"Received Zigbee message from 'KlimaTuya01', type 'attributeReport', cluster 'msIlluminanceMeasurement', data '{\"61441\":50}' from endpoint 1 with groupID 0"}
Client mosqsub|846-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/logging', ... (515 bytes))
{"level":"info","message":"MQTT publish: topic 'zigbee2mqtt/KlimaTuya01', payload '{\"battery\":100,\"device\":{\"applicationVersion\":67,\"dateCode\":\"\",\"friendlyName\":\"KlimaTuya01\",\"hardwareVersion\":1,\"ieeeAddr\":\"0x842e14fffee547f2\",\"manufacturerID\":4098,\"manufacturerName\":\"_TZ3000_qaaysllp\",\"model\":\"TS0201\",\"networkAddress\":55096,\"powerSource\":\"Battery\",\"stackVersion\":0,\"type\":\"EndDevice\",\"zclVersion\":3},\"illuminance\":null,\"illuminance_lux\":null,\"linkquality\":24}'"}
Client mosqsub|846-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/KlimaTuya01', ... (379 bytes))
{"battery":100,"device":{"applicationVersion":67,"dateCode":"","friendlyName":"KlimaTuya01","hardwareVersion":1,"ieeeAddr":"0x842e14fffee547f2","manufacturerID":4098,"manufacturerName":"_TZ3000_qaaysllp","model":"TS0201","networkAddress":55096,"powerSource":"Battery","stackVersion":0,"type":"EndDevice","zclVersion":3},"illuminance":null,"illuminance_lux":null,"linkquality":24}
Client mosqsub|846-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/logging', ... (196 bytes))
{"level":"debug","message":"Received Zigbee message from 'KlimaTuya01', type 'raw', cluster '57346', data '{\"data\":[8,37,10,10,208,41,40,0],\"type\":\"Buffer\"}' from endpoint 1 with groupID 0"}
Client mosqsub|846-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/logging', ... (168 bytes))
{"level":"debug","message":"No converter available for 'TS0201' with cluster '57346' and type 'raw' and data '{\"data\":[8,37,10,10,208,41,40,0],\"type\":\"Buffer\"}'"}
Client mosqsub|846-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/logging', ... (196 bytes))
{"level":"debug","message":"Received Zigbee message from 'KlimaTuya01', type 'raw', cluster '57346', data '{\"data\":[8,38,10,11,208,41,10,0],\"type\":\"Buffer\"}' from endpoint 1 with groupID 0"}
Client mosqsub|846-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/logging', ... (168 bytes))
{"level":"debug","message":"No converter available for 'TS0201' with cluster '57346' and type 'raw' and data '{\"data\":[8,38,10,11,208,41,10,0],\"type\":\"Buffer\"}'"}
Client mosqsub|846-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/logging', ... (196 bytes))
{"level":"debug","message":"Received Zigbee message from 'KlimaTuya01', type 'raw', cluster '57346', data '{\"data\":[8,39,10,13,208,41,90,0],\"type\":\"Buffer\"}' from endpoint 1 with groupID 0"}
Client mosqsub|846-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/logging', ... (168 bytes))
{"level":"debug","message":"No converter available for 'TS0201' with cluster '57346' and type 'raw' and data '{\"data\":[8,39,10,13,208,41,90,0],\"type\":\"Buffer\"}'"}
Client mosqsub|846-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/logging', ... (196 bytes))
{"level":"debug","message":"Received Zigbee message from 'KlimaTuya01', type 'raw', cluster '57346', data '{\"data\":[8,40,10,14,208,41,20,0],\"type\":\"Buffer\"}' from endpoint 1 with groupID 0"}
Client mosqsub|846-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/logging', ... (168 bytes))
{"level":"debug","message":"No converter available for 'TS0201' with cluster '57346' and type 'raw' and data '{\"data\":[8,40,10,14,208,41,20,0],\"type\":\"Buffer\"}'"}
Client mosqsub|846-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/logging', ... (192 bytes))
{"level":"debug","message":"Received Zigbee message from 'KlimaTuya01', type 'raw', cluster '57346', data '{\"data\":[8,41,10,6,208,48,2],\"type\":\"Buffer\"}' from endpoint 1 with groupID 0"}
Client mosqsub|846-openhab sending PINGREQ
Client mosqsub|846-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/logging', ... (164 bytes))
{"level":"debug","message":"No converter available for 'TS0201' with cluster '57346' and type 'raw' and data '{\"data\":[8,41,10,6,208,48,2],\"type\":\"Buffer\"}'"}
Client mosqsub|846-openhab received PINGRESP
Client mosqsub|846-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/logging', ... (193 bytes))
{"level":"debug","message":"Received Zigbee message from 'KlimaTuya01', type 'raw', cluster '57346', data '{\"data\":[8,42,10,15,208,48,2],\"type\":\"Buffer\"}' from endpoint 1 with groupID 0"}
Client mosqsub|846-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/logging', ... (165 bytes))
{"level":"debug","message":"No converter available for 'TS0201' with cluster '57346' and type 'raw' and data '{\"data\":[8,42,10,15,208,48,2],\"type\":\"Buffer\"}'"}
Client mosqsub|846-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/logging', ... (193 bytes))
{"level":"debug","message":"Received Zigbee message from 'KlimaTuya01', type 'raw', cluster '57346', data '{\"data\":[8,43,10,16,208,32,0],\"type\":\"Buffer\"}' from endpoint 1 with groupID 0"}
Client mosqsub|846-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/logging', ... (165 bytes))
{"level":"debug","message":"No converter available for 'TS0201' with cluster '57346' and type 'raw' and data '{\"data\":[8,43,10,16,208,32,0],\"type\":\"Buffer\"}'"}
Client mosqsub|846-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/logging', ... (197 bytes))
{"level":"debug","message":"Received Zigbee message from 'KlimaTuya01', type 'attributeReport', cluster 'msIlluminanceMeasurement', data '{\"measuredValue\":28574}' from endpoint 1 with groupID 0"}
Client mosqsub|846-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/bridge/logging', ... (515 bytes))
{"level":"info","message":"MQTT publish: topic 'zigbee2mqtt/KlimaTuya01', payload '{\"battery\":100,\"device\":{\"applicationVersion\":67,\"dateCode\":\"\",\"friendlyName\":\"KlimaTuya01\",\"hardwareVersion\":1,\"ieeeAddr\":\"0x842e14fffee547f2\",\"manufacturerID\":4098,\"manufacturerName\":\"_TZ3000_qaaysllp\",\"model\":\"TS0201\",\"networkAddress\":55096,\"powerSource\":\"Battery\",\"stackVersion\":0,\"type\":\"EndDevice\",\"zclVersion\":3},\"illuminance\":28574,\"illuminance_lux\":720,\"linkquality\":27}'"}
Client mosqsub|846-openhab received PUBLISH (d0, q0, r0, m0, 'zigbee2mqtt/KlimaTuya01', ... (379 bytes))
{"battery":100,"device":{"applicationVersion":67,"dateCode":"","friendlyName":"KlimaTuya01","hardwareVersion":1,"ieeeAddr":"0x842e14fffee547f2","manufacturerID":4098,"manufacturerName":"_TZ3000_qaaysllp","model":"TS0201","networkAddress":55096,"powerSource":"Battery","stackVersion":0,"type":"EndDevice","zclVersion":3},"illuminance":28574,"illuminance_lux":720,"linkquality":27}`

**Reply by unstable09** [reply]
It looks like I have the same device, also reporting as:
`Debug Received Zigbee message from '0x847127fffefe817f', type 'readResponse', cluster 'genBasic', data '{"modelId":"TS0201"}' from endpoint 1 with groupID null
Debug Received Zigbee message from '0x847127fffefe817f', type 'readResponse', cluster 'genBasic', data '{"manufacturerName":"_TZ3000_qaaysllp"}' from endpoint 1 with groupID null
Debug Received Zigbee message from '0x847127fffefe817f', type 'readResponse', cluster 'genBasic', data '{"powerSource":3}' from endpoint 1 with groupID null
Debug Received Zigbee message from '0x847127fffefe817f', type 'readResponse', cluster 'genBasic', data '{"zclVersion":3}' from endpoint 1 with groupID null
Debug Received Zigbee message from '0x847127fffefe817f', type 'readResponse', cluster 'genBasic', data '{"appVersion":68}' from endpoint 1 with groupID null
Debug Received Zigbee message from '0x847127fffefe817f', type 'readResponse', cluster 'genBasic', data '{"stackVersion":0}' from endpoint 1 with groupID null
Debug Received Zigbee message from '0x847127fffefe817f', type 'readResponse', cluster 'genBasic', data '{"hwVersion":1}' from endpoint 1 with groupID null
Debug Received Zigbee message from '0x847127fffefe817f', type 'readResponse', cluster 'genBasic', data '{"dateCode":""}' from endpoint 1 with groupID null
Debug Received Zigbee message from '0x847127fffefe817f', type 'readResponse', cluster 'genBasic', data '{}' from endpoint 1 with groupID null`

I added new device and suppert for illuminance, but I also stuck on buffer with raw type... Any luck with implementation? Is there some documentation about how to add device with raw data type?

Me log when I try to add device with my custom device file:

`Debug Received MQTT message on 'zigbee2mqtt/bridge/request/device/remove' with data '{"block":false,"force":true,"id":"0x847127fffefe817f","transaction":"wghfg-1"}'
Info Removing device '0x847127fffefe817f' (block: false, force: true)
Debug Clearing Home Assistant discovery topic for '0x847127fffefe817f'
Info MQTT publish: topic 'homeassistant/sensor/0x847127fffefe817f/illuminance_lux/config', payload 'null'
Info MQTT publish: topic 'homeassistant/sensor/0x847127fffefe817f/linkquality/config', payload 'null'
Info MQTT publish: topic 'zigbee2mqtt/0x847127fffefe817f', payload ''
Info Successfully removed device '0x847127fffefe817f' (block: false, force: true)
Info MQTT publish: topic 'zigbee2mqtt/bridge/response/device/remove', payload '{"data":{"block":false,"force":true,"id":"0x847127fffefe817f"},"status":"ok","transaction":"wghfg-1"}'
Debug Received MQTT message on 'homeassistant/sensor/0x847127fffefe817f/illuminance_lux/config' with data ''
Debug Received MQTT message on 'homeassistant/sensor/0x847127fffefe817f/linkquality/config' with data ''
Info Device '0x847127fffefe817f' joined
Info MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0x847127fffefe817f","ieee_address":"0x847127fffefe817f"},"type":"device_joined"}'
Info Starting interview of '0x847127fffefe817f'
Info MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0x847127fffefe817f","ieee_address":"0x847127fffefe817f","status":"started"},"type":"device_interview"}'
Info MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":{"friendly_name":"0x847127fffefe817f"},"type":"device_connected"}'
Info MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"interview_started","meta":{"friendly_name":"0x847127fffefe817f"},"type":"pairing"}'
Debug Received Zigbee message from '0x847127fffefe817f', type 'attributeReport', cluster 'msIlluminanceMeasurement', data '{"61441":50}' from endpoint 1 with groupID null
Debug No converter available for 'TS0201' with cluster 'msIlluminanceMeasurement' and type 'attributeReport' and data '{"61441":50}'
Debug Received Zigbee message from '0x847127fffefe817f', type 'raw', cluster '57346', data '{"data":[8,81,10,10,208,41,40,0],"type":"Buffer"}' from endpoint 1 with groupID null
Debug Skipping message, definition is undefined and still interviewing
Debug Received Zigbee message from '0x847127fffefe817f', type 'readResponse', cluster 'genBasic', data '{"modelId":"TS0201"}' from endpoint 1 with groupID null
Debug Received Zigbee message from '0x847127fffefe817f', type 'readResponse', cluster 'genBasic', data '{"manufacturerName":"_TZ3000_qaaysllp"}' from endpoint 1 with groupID null
Debug Received Zigbee message from '0x847127fffefe817f', type 'readResponse', cluster 'genBasic', data '{"powerSource":3}' from endpoint 1 with groupID null
Debug Received Zigbee message from '0x847127fffefe817f', type 'readResponse', cluster 'genBasic', data '{"zclVersion":3}' from endpoint 1 with groupID null
Debug Received Zigbee message from '0x847127fffefe817f', type 'readResponse', cluster 'genBasic', data '{"appVersion":68}' from endpoint 1 with groupID null
Debug Received Zigbee message from '0x847127fffefe817f', type 'readResponse', cluster 'genBasic', data '{"stackVersion":0}' from endpoint 1 with groupID null
Debug Received Zigbee message from '0x847127fffefe817f', type 'readResponse', cluster 'genBasic', data '{"hwVersion":1}' from endpoint 1 with groupID null
Debug Received Zigbee message from '0x847127fffefe817f', type 'readResponse', cluster 'genBasic', data '{"dateCode":""}' from endpoint 1 with groupID null
Debug Received Zigbee message from '0x847127fffefe817f', type 'readResponse', cluster 'genBasic', data '{}' from endpoint 1 with groupID null
Info Successfully interviewed '0x847127fffefe817f', device has successfully been paired
Info Device '0x847127fffefe817f' is supported, identified as: Tuya Temperature, humidity and illuminance sensor (TS0201_Sensor)
Info MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"definition":{"description":"Temperature, humidity and illuminance sensor","exposes":[{"access":1,"description":"Measured illuminance in lux","name":"illuminance_lux","property":"illuminance_lux","type":"numeric","unit":"lx"},{"access":1,"description":"Link quality (signal strength)","name":"linkquality","property":"linkquality","type":"numeric","unit":"lqi","value_max":255,"value_min":0}],"model":"TS0201_Sensor","supports_ota":false,"vendor":"Tuya"},"friendly_name":"0x847127fffefe817f","ieee_address":"0x847127fffefe817f","status":"successful","supported":true},"type":"device_interview"}'
Info MQTT publish: topic 'homeassistant/sensor/0x847127fffefe817f/illuminance_lux/config', payload '{"availability":[{"topic":"zigbee2mqtt/bridge/state"}],"device":{"identifiers":["zigbee2mqtt_0x847127fffefe817f"],"manufacturer":"Tuya","model":"Temperature, humidity and illuminance sensor (TS0201_Sensor)","name":"0x847127fffefe817f","sw_version":"Zigbee2MQTT 1.21.0"},"device_class":"illuminance","json_attributes_topic":"zigbee2mqtt/0x847127fffefe817f","name":"0x847127fffefe817f illuminance lux","state_class":"measurement","state_topic":"zigbee2mqtt/0x847127fffefe817f","unique_id":"0x847127fffefe817f_illuminance_lux_zigbee2mqtt","unit_of_measurement":"lx","value_template":"{{ value_json.illuminance_lux }}"}'
Info MQTT publish: topic 'homeassistant/sensor/0x847127fffefe817f/linkquality/config', payload '{"availability":[{"topic":"zigbee2mqtt/bridge/state"}],"device":{"identifiers":["zigbee2mqtt_0x847127fffefe817f"],"manufacturer":"Tuya","model":"Temperature, humidity and illuminance sensor (TS0201_Sensor)","name":"0x847127fffefe817f","sw_version":"Zigbee2MQTT 1.21.0"},"enabled_by_default":false,"icon":"mdi:signal","json_attributes_topic":"zigbee2mqtt/0x847127fffefe817f","name":"0x847127fffefe817f linkquality","state_class":"measurement","state_topic":"zigbee2mqtt/0x847127fffefe817f","unique_id":"0x847127fffefe817f_linkquality_zigbee2mqtt","unit_of_measurement":"lqi","value_template":"{{ value_json.linkquality }}"}'
Info MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"interview_successful","meta":{"description":"Temperature, humidity and illuminance sensor","friendly_name":"0x847127fffefe817f","model":"TS0201_Sensor","supported":true,"vendor":"Tuya"},"type":"pairing"}'
Debug Received MQTT message on 'homeassistant/sensor/0x847127fffefe817f/illuminance_lux/config' with data '{"availability":[{"topic":"zigbee2mqtt/bridge/state"}],"device":{"identifiers":["zigbee2mqtt_0x847127fffefe817f"],"manufacturer":"Tuya","model":"Temperature, humidity and illuminance sensor (TS0201_Sensor)","name":"0x847127fffefe817f","sw_version":"Zigbee2MQTT 1.21.0"},"device_class":"illuminance","json_attributes_topic":"zigbee2mqtt/0x847127fffefe817f","name":"0x847127fffefe817f illuminance lux","state_class":"measurement","state_topic":"zigbee2mqtt/0x847127fffefe817f","unique_id":"0x847127fffefe817f_illuminance_lux_zigbee2mqtt","unit_of_measurement":"lx","value_template":"{{ value_json.illuminance_lux }}"}'
Debug Received MQTT message on 'homeassistant/sensor/0x847127fffefe817f/linkquality/config' with data '{"availability":[{"topic":"zigbee2mqtt/bridge/state"}],"device":{"identifiers":["zigbee2mqtt_0x847127fffefe817f"],"manufacturer":"Tuya","model":"Temperature, humidity and illuminance sensor (TS0201_Sensor)","name":"0x847127fffefe817f","sw_version":"Zigbee2MQTT 1.21.0"},"enabled_by_default":false,"icon":"mdi:signal","json_attributes_topic":"zigbee2mqtt/0x847127fffefe817f","name":"0x847127fffefe817f linkquality","state_class":"measurement","state_topic":"zigbee2mqtt/0x847127fffefe817f","unique_id":"0x847127fffefe817f_linkquality_zigbee2mqtt","unit_of_measurement":"lqi","value_template":"{{ value_json.linkquality }}"}'
Debug Received Zigbee message from '0x847127fffefe817f', type 'raw', cluster '57346', data '{"data":[8,82,10,11,208,41,10,0],"type":"Buffer"}' from endpoint 1 with groupID null
Debug No converter available for 'TS0201_Sensor' with cluster '57346' and type 'raw' and data '{"data":[8,82,10,11,208,41,10,0],"type":"Buffer"}'
Debug Received Zigbee message from '0x847127fffefe817f', type 'raw', cluster '57346', data '{"data":[8,83,10,13,208,41,90,0],"type":"Buffer"}' from endpoint 1 with groupID null
Debug No converter available for 'TS0201_Sensor' with cluster '57346' and type 'raw' and data '{"data":[8,83,10,13,208,41,90,0],"type":"Buffer"}'
Debug Received Zigbee message from '0x847127fffefe817f', type 'raw', cluster '57346', data '{"data":[8,84,10,14,208,41,20,0],"type":"Buffer"}' from endpoint 1 with groupID null
Debug No converter available for 'TS0201_Sensor' with cluster '57346' and type 'raw' and data '{"data":[8,84,10,14,208,41,20,0],"type":"Buffer"}'
Debug Received Zigbee message from '0x847127fffefe817f', type 'raw', cluster '57346', data '{"data":[8,85,10,6,208,48,2],"type":"Buffer"}' from endpoint 1 with groupID null
Debug No converter available for 'TS0201_Sensor' with cluster '57346' and type 'raw' and data '{"data":[8,85,10,6,208,48,2],"type":"Buffer"}'
Debug Received Zigbee message from '0x847127fffefe817f', type 'raw', cluster '57346', data '{"data":[8,86,10,15,208,48,2],"type":"Buffer"}' from endpoint 1 with groupID null
Debug No converter available for 'TS0201_Sensor' with cluster '57346' and type 'raw' and data '{"data":[8,86,10,15,208,48,2],"type":"Buffer"}'
Debug Received Zigbee message from '0x847127fffefe817f', type 'raw', cluster '57346', data '{"data":[8,87,10,16,208,32,0],"type":"Buffer"}' from endpoint 1 with groupID null
Debug No converter available for 'TS0201_Sensor' with cluster '57346' and type 'raw' and data '{"data":[8,87,10,16,208,32,0],"type":"Buffer"}'
Debug Received Zigbee message from '0x847127fffefe817f', type 'attributeReport', cluster 'msIlluminanceMeasurement', data '{"measuredValue":17244}' from endpoint 1 with groupID null
Info MQTT publish: topic 'zigbee2mqtt/0x847127fffefe817f', payload '{"illuminance":17244,"illuminance_lux":53,"linkquality":247}'
Debug Received Zigbee message from '0x847127fffefe817f', type 'read', cluster 'genTime', data '["localTime"]' from endpoint 1 with groupID null
Debug Saving state to file /config/zigbee2mqtt/state.json
`
